pkg/csi/service/common/vsphereutil.go

```diff
@@ -645,3 +646,33 @@ func isExpansionRequired(ctx context.Context, volumeID string, requestedSize int
 
 	return currentSize < requestedSize, nil
 }
+
+// CreateSnapshotUtil is the helper function to create CNS snapshot for given volumeId
+func CreateSnapshotUtil(ctx context.Context, manager *Manager, volumeID string, desc string) (string, *time.Time, error) {
```

Comment:
What is expected in "desc string"?  Please add comments.
Also the returned "string" is "volume id"+"snapshot id"?  Please add comments.

Reply:
Can you add an example of desc in the comments?